src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken({banned: true});
+  }, [props.token]);
+
+  const verifiedAlert = () => {
+    if (!token.verified) {
```

Comment:
`this.props.token.verified`?

Reply:
I think it will not be reactive that way.